src/scan-dirs.ts

```diff
@@ -61,6 +61,19 @@ export async function scanExports (filepath: string) {
       if (exp.name) {
         imports.push({ name: exp.name, as: exp.name, from: filepath })
       }
+    } else if (exp.type === 'star' && exp.specifier) {
+      let subfile = exp.specifier
+      if (!extname(subfile)) {
+        subfile = `${subfile}.ts` // this is not ideal
+      }
+      const subfilepath = join(dirname(filepath), subfile)
+      const subimports = await scanExports(subfilepath)
+
+      if (exp.name) {
+        imports.push({ name: '*', as: exp.name, from: subfilepath })
+      } else {
+        imports.push(...subimports)
```

Comment:
`subimports` may not be required. `subfilepath` is all that would be required I believe (and should also eliminate the need of `extname`, but I'm open to suggestions.